pages/profile/[username].js

```diff
@@ -183,9 +183,9 @@ function Profile() {
 									<Tab.Pane eventKey="hackathons">
 										<div>
 											<Row>
-												{ userRequest.user.teams && userRequest.user.teams.length
+												{userRequest.user.teams && userRequest.user.teams.length
 													? userRequest.user.teams.map(({ hackathon, id }) => (
-														<Col sm={6} className="pb-3" key={hackathon.slug} key={id}>
+														<Col sm={6} className="pb-3" key={hackathon.slug + "-" + id}>
```

Comment:
In this case, we can leave it to only using the `slug` or `id` as both these fields are unique for a hackathon.